internal/mode/static/nginx/config/servers.go

```diff
@@ -544,8 +538,8 @@ func createPath(rule dataplane.PathRule) string {
 	}
 }
 
-func createPathForMatch(path string, pathType dataplane.PathType, routeIdx int) string {
-	return fmt.Sprintf("%s_%s_route%d", path, pathType, routeIdx)
+func createPathForMatch(ruleIdx, routeIdx int) string {
```

Comment:
Let's rename this function to `createNamedLocationPath`, or we could remove the function and create the path in the `initializeInternalLocation` function.

Reply:
sounds good, updating